docs/components/icons/index.html

```diff
@@ -75,12 +75,18 @@ <h2 id="file-icons">File Icons</h2>
 
   <section id="vue-iconDemo">
     <h2 id="available-icons">Icons</h2>
+    <p>
+      <a href="downloads/helix-ui-{{VERSION}}-icons.tgz">
+        <hx-icon type="download"></hx-icon>
+        Download All Icons (SVG)
+      </a>
+    </p>
     <div v-cloak>
       <div class="hxRow">
         <div class="hxCol hxSpan-12-xs hxSpan-6-md hxSpan-4-xl">
           <p>
             <hx-search
-              :placeholder="`Search ${icons.length} icons for...`"
```

Comment:
Vue throws an error on this line, because IE doesn't support template literals. Moved it into the JS so that babel can transpile to ES5-compatible code.